_delphi_utils_python/data_proc/geomap/geo_data_proc.py

```diff
@@ -400,7 +400,14 @@ def create_fips_population_table():
     df_pr = df_pr[~df_pr["fips"].isin(census_pop["fips"])]
     census_pop_pr = pd.concat([census_pop, df_pr])
 
-    census_pop_pr.to_csv(join(OUTPUT_DIR, FIPS_POPULATION_OUT_FILENAME), index=False)
+    # Filled from https://www.census.gov/data/tables/2010/dec/2010-island-areas.html
+    territories_pop = pd.DataFrame({
+        "fips": ["60010", "60020", "60030", "60040", "60050", "66010", "78010", "78020", "78030", "69085", "69100", "69110", "69120"],
+        "pop": [23030, 1143, 0, 17, 31329, 159358, 50601, 4170, 51634, 0, 2527, 48220, 3136]
```

Comment:
The sources for the FIPS county codes to names can be found [here](https://en.wikipedia.org/wiki/List_of_United_States_FIPS_codes_by_county#cite_note-CensusT-15) and cross-referenced [here](https://www.nrcs.usda.gov/wps/portal/nrcs/detail/national/home/?cid=nrcs143_013697) and [here](http://datadictionary.naaccr.org/default.aspx?c=11&Version=21#Virgin%20Islands%20of%20the%20United%20States). The population sources can be found at the Census [here](https://www.census.gov/data/tables/2010/dec/2010-island-areas.html). I double checked the numbers and all look good :+1:

Reply:
While there are a couple other territories that have close aid relationships with the United States such as Marshall Islands, the Republic of Palau, and Micronesia, our data sources do not appear to report on these at the moment. Adding support for those can be done as needed in a separate PR.